analyzer/queries/queries.go

```diff
@@ -808,4 +834,104 @@ var (
       source_files = $5
     WHERE
       runtime = $1 AND contract_address = $2`
+
+	RuntimeEvmVerifiedContractTxs = `
+    WITH abi_contracts AS (
+      SELECT 
+        runtime,
+        contract_address AS addr,
+        abi
+      FROM chain.evm_contracts
+      WHERE
+        runtime = $1 AND verification_info_downloaded_at IS NOT NULL
+    )
+    SELECT 
+      abi_contracts.addr,
+      abi_contracts.abi,
+      txs.tx_hash,
+      txs.body->'data',
+      txs.error_message_raw,
+    FROM abi_contracts 
+    JOIN chain.runtime_transactions as txs ON
+      txs.runtime = abi_contracts.runtime AND
+      txs.to = abi_contracts.addr AND
+      txs.method = 'evm.Call' -- note: does not include evm.Create txs
+    WHERE
+      runtime = $1 AND
+      processed_at IS NULL
+    ORDER BY addr
+    LIMIT $2`
+
+	RuntimeEvmVerifiedContractTxsCount = `
+    WITH abi_contracts AS (
+      SELECT 
+        runtime,
+        contract_address AS addr
+      FROM chain.evm_contracts
+      WHERE
+        runtime = $1 AND verification_info_downloaded_at IS NOT NULL
+    )
+    SELECT COUNT(*)
+    FROM abi_contracts 
+    JOIN chain.runtime_transactions as txs ON
+      txs.runtime = abi_contracts.runtime AND
+      txs.to = abi_contracts.addr AND
+      txs.method = 'evm.Call' -- note: does not include evm.Create txs
+    WHERE
+      runtime = $1 AND -- todo: redundant; confirm that it does not improve performance and remove
```

Comment:
This query (as a whole) feels like it could be very expensive. At the same time, it is not very important; it's just for our internal grafana dashboard.
Can you try running it manually on the prod db? If it takes more than a second or maybe two, I'd say that's undue DB load. In that case, I suggest we use the same approach we use for pagination, and be ok with clipping the true count. So, to run something like
```go
db.Query("SELECT count(*) FROM (" + RuntimeEvmVerifiedContractTxs + ") foo"`, 10000)
// ^ 10000 is the limit for the inner query
```

Reply:
good eye - turns out even the original queries used in `GetItems` are too slow for prod emerald bc of the join on the events table... will experiment further

Reply:
Added the clipped query for abi events; transactions query was fast enough